data/generic_nullable_vector.go

```diff
@@ -10,7 +10,17 @@ func newNullablegenVector(n int) *nullablegenVector {
 }
 
 func (v *nullablegenVector) Set(idx int, i interface{}) {
-	(*v)[idx] = i.(*gen)
+	if i == nil {
```

Comment:
This is good, this should make it so if you pass nil, it doesn't have to be a typed nil.